Unittest.py

```diff
@@ -202,7 +372,7 @@ def test_fuzzer(self):
             'randomize_settings': True,
             'compress_rom': "None",
             'create_spoiler': True,
-            'output_file': os.path.join(output_dir, 'fuzz-%d' % i),
```

Comment:
This isn't necessary outside of json (and I usually prefer including trailing commas where possible, because adding new entries at the end won't generate additional diffs in future changes).

Reply:
Not entirely sure what you mean here? Just seems a comma was removed which was likely just my IDE being too aggressive about code format and I can restore that. The rest was already there as far as I can tell.